engine/src/main/java/org/destinationsol/game/chunk/ChunkFiller.java

```diff
@@ -176,19 +179,15 @@ private void fillEnemies(SolGame game, RemoveController removeController, ShipCo
         }
 
         for (int i = 0; i < enemyCount; i++) {
-            Vector2 enemyPosition = getFreeRndPos(game, chunkCenter);
-            FarShip ship = buildSpaceEnemy(game, enemyPosition, removeController, enemyConfig);
-            if (ship != null) {
+            Optional<Vector2> enemyPosition = getFreeRndPos(game, chunkCenter);
+            enemyPosition.ifPresent(position -> {
```

Comment:
I'd prefer keeping the name of the introduced lambda parameter similar (or same if the compiler allows it) to the `Optional` variable name to retain context and clarity.